prow/config/config.go

```diff
@@ -717,6 +718,23 @@ func setPeriodicDecorationDefaults(c *Config, ps *Periodic) {
 	}
 }
 
+func finalizePresubmits(c *Config, presubmits []Presubmit) error {
+	setPresubmitDecorationDefaults(c, presubmits)
```

Comment:
We should only do this `if c.decorationRequested()` and we should only check that once since it is somewhat expensive.